### PR TITLE
Fix misc sound bugs

### PR DESCRIPTION
**Pull request by TheRealRavager**
Fixes issues #709 and #711

### CODE REVIEW

**Comment by TheRealRavager** [comment]
Just realised, values for stacking adsr and instrument sounds need to be changed. Added WIP to title

File: public/externalLibs/sound/sounds.js [hunk 1]
```diff
@@ -308,14 +306,18 @@ var _safeaudio = null;
...
+        throw new Error("play: audio system still playing previous sound");
+    } else {
+        // Discretize the input sound
+        var data = discretize(get_wave(sound), get_duration(sound));
+        _safeaudio = raw_to_audio(data);
+
+        _safeaudio.addEventListener('ended', stop);
+        _safeaudio.play();
+        _safeplaying = true;
+        return sound;
```
Comment (geshuming) [inline public/externalLibs/sound/sounds.js]: The doc says it's supposed to return `undefined`, but this isn't undefined.

**Comment by geshuming** [comment]
Can you rebase this PR too, it's getty messy

**Comment by TheRealRavager** [comment]
> Can you rebase this PR too, it's getty messy

Rebased

File: public/externalLibs/sound/sounds.js (hunk 1 above)
Comment (TheRealRavager) [reply public/externalLibs/sound/sounds.js]: The PR is supposed to change the specs for some functions (e.g. ADSR now accepts ratios instead of durations)

Comment (TheRealRavager) [reply public/externalLibs/sound/sounds.js]: Where can I update the doc?

Comment (geshuming) [reply public/externalLibs/sound/sounds.js]: The jsdoc of this function

Comment (TheRealRavager) [reply public/externalLibs/sound/sounds.js]: Changed the jsdoc. Does the official source documentation pull the info from the academy frontend or is it static?

**Pull request merged**